zkevm-circuits/src/evm_circuit/execution/memory.rs

```diff
@@ -203,11 +252,22 @@ impl<F: Field> ExecutionGadget<F> for MemoryGadget<F> {
             block.rws[step.rw_indices[3]].memory_word_value()
         };
 
+        // TODO: rm.
+        println!("address: {:?}", address);
+        println!("value (LE):  {:?}", value.to_le_bytes());
+        println!("value_left:  {:?}", value_left.to_le_bytes());
+        println!("value_right: {:?}", value_right.to_le_bytes());
+
+        // TODO: updated values for MSTORE.
         self.value_left
```

Comment:
just confirm: for mload, value == value_pre for both left and right . for mstore/mstore8, value != value_prev ?

Reply:
Correct.

Reply:
would you like to add it in this PR ? plan to merge this PR after added.

Reply:
I will add this in another PR.

Reply:
ok, overall lgtm, will merge it.